### DISCUSSION TITLE
[Feature]:  fabric-with-gestures-v5 typeScript not support?

### DISCUSSION THREAD

**Discussion by zj91zj** [fabricjs/fabric.js#9821] [Q&A]
### CheckList

- [X] I agree to follow this project's Code of Conduct
- [X] I have read and followed the [Contributing Guide](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md)
- [X] I have searched and referenced existing issues, feature requests and discussions
- [X] I am filing a **FEATURE** request.

### Description

Module "./fabric-with-gestures-v5" does not exist in container

### Current State

Module "./fabric-with-gestures-v5" does not exist in container

**Comment by asturur** [comment]
Is unclear what are you asking
